drivers/scheduler/k8s/k8s.go

```diff
@@ -3872,16 +3872,10 @@ func (k *K8s) ValidateVolumes(ctx *scheduler.Context, timeout, retryInterval tim
 			autopilotLabels["name"] = "autopilot"
 			autopilotPods, err := k8sCore.GetPods(autopilotDefaultNamespace, autopilotLabels)
```

Comment:
Can we find any there is a function that gets Portworx namespace or write one and use it here instead of hardcoded `kube-system`, as it is not default namespace anymore

Reply:
> Can we find any there is a function that gets Portworx namespace or write one and use it here instead of hardcoded `kube-system`, as it is not default namespace anymore

Addressed now